### PR TITLE
windows-applet: group alt-tab options, remove reduced-resources option

### PR DESCRIPTION
**Pull request by mbkma**
in a way it makes more sense. The alt-tab options are now grouped together. The "Compositing Manager" tab is now the "Performance" tab, which is better because there can be added more options in the future such as "disable animations".

Screenshot at 2021-03-28 11-23-49
Screenshot at 2021-03-28 11-24-17
Screenshot at 2021-03-28 11-24-34

### CODE REVIEW

**Comment by raveit65** [comment]
Not sure if it is really possible to reduce/increase performance with using the compositor manager nowadays with modern CPUs and GPUs.
I would say it doesn't matter from performance point of view if you use compositor or not :)
Maybe with a proc from last century or the zero years.
So i am not really sure if `performance` is the right name for this tab.
But i am not against it if other member of the team thinks the new name makes sense.

**Comment by raveit65** [comment]
```
[rave@mother mate-control-center]$ git checkout -b mbkma-reorder master
Zu neuem Branch 'mbkma-reorder' gewechselt
[rave@mother mate-control-center]$ LANG=C
[rave@mother mate-control-center]$ git pull https://github.com/mbkma/mate-control-center.git reorder
From https://github.com/mbkma/mate-control-center
 * branch              reorder    -> FETCH_HEAD
fatal: Not possible to fast-forward, aborting.
[rave@mother mate-control-center]$ 
```
I wanna bet your master branch isn't synced with our master.

**Review by raveit65 — APPROVED** [review]
LGTM,

**Comment by mbkma** [comment]
I just did a `git rebase upstream/master`. @raveit65 I am not sure about the Performance tab as well. Btw. in a virtual machine I have experienced the following glitches (black window border, sometimes the grid appears twice) with the reduced resources options 
Screenshot at 2021-04-17 22-45-48
Maybe its better to remove the reduced resources option?

**Comment by raveit65** [comment]
I recall there was a weird bugreport another repo. After a long debuging we could mark `reduced resources` option as culprit, ........some years ago. But i can't remember which report it was.
So, i prefer and agree with removing this option from GUI.
In result there is no reason to rename this tab, or not?

**Comment by lukefromdc** [comment]
Do we have users (say on Raspberry Pi?) who need the reduced resources option to keep window resizing responsive or not run out of VRAM or system RAM when it is already nearly full?

**Comment by mbkma** [comment]
Yes I have a raspberry pi 3b+ and the reduced resources option does help a little bit but not too much (subjectively, without real testing). I eventually disabled it on my pi, because of the glitches I get with this option enabled. The option was removed from metacity a year ago: https://gitlab.gnome.org/GNOME/metacity/-/commit/aab0e5f7c390f2ba5c95498e5d28a88467b7f5f1

**Comment by mbkma** [comment]
Screenshot at 2021-04-18 09-30-07
Screenshot at 2021-04-18 09-30-28

**Comment by raveit65** [comment]
```
[rave@mother mate-control-center]$ git checkout -b mbkma-reorder master
Zu neuem Branch 'mbkma-reorder' gewechselt
[rave@mother mate-control-center]$ git pull https://github.com/mbkma/mate-control-center.git reorder
remote: Enumerating objects: 11, done.
remote: Counting objects: 100% (11/11), done.
remote: Compressing objects: 100% (1/1), done.
remote: Total 6 (delta 5), reused 6 (delta 5), pack-reused 0
Entpacke Objekte: 100% (6/6), 769 Bytes | 109.00 KiB/s, fertig.
Von https://github.com/mbkma/mate-control-center
 * branch              reorder    -> FETCH_HEAD
fatal: Vorspulen nicht möglich, breche ab.
```
OMG

**Comment by mbkma** [comment]
@raveit65 done

**Review by raveit65 — COMMENTED** [review]
LGTM,
reeeeeeeeady toooooooo ruuuuuuuuumble

Ali

File: capplets/windows/mate-window-properties.c [hunk 1]
```diff
@@ -124,11 +122,8 @@ update_sensitivity (void)
 
     gtk_widget_set_sensitive (compositing_fast_alt_tab_checkbutton,
                               g_settings_get_boolean (marco_settings, MARCO_COMPOSITING_MANAGER_KEY));
-    gtk_widget_set_sensitive (enable_tiling_checkbutton,
-                              !g_settings_get_boolean (marco_settings, MARCO_REDUCED_RESOURCES_KEY));
```
Comment (rbuj) [inline capplets/windows/mate-window-properties.c:128]: Why did you remove this when this preference is currently in use in marco?
```C
          !meta_prefs_get_reduced_resources ())
```


https://github.com/mate-desktop/marco/blob/aebd99901c99c49944bc514020f4517c545111c7/src/core/window.c#L1503

Comment (rbuj) [reply capplets/windows/mate-window-properties.c:128]: PR #646

Comment (mbkma) [reply capplets/windows/mate-window-properties.c:128]: see https://github.com/mate-desktop/mate-control-center/pull/641#issuecomment-821887597

The setting can still be enabled via dconf-editor or command line. I would vote for not expose a broken feature in mcc.

Also see

> Wireframe mode has not been updated/fixed to work with CSD windows
> and/or to exclude shadow/resize area from SSD windows. Also quickly
> testing with and without compositor wireframe sometimes leaves
> parts of frame on screen.
> 
> Considering that there has not been any bug report about broken
> wireframe mode I think it is time to remove it instead of wasting
> time trying to fix something that nobody is using.

(https://gitlab.gnome.org/GNOME/metacity/-/commit/aab0e5f7c390f2ba5c95498e5d28a88467b7f5f1)

Comment (rbuj) [reply capplets/windows/mate-window-properties.c:128]: ok, don't worry
